src/main/scala/com/microsoft/partnercatalyst/fortis/spark/Pipeline.scala

```diff
@@ -0,0 +1,100 @@
+package com.microsoft.partnercatalyst.fortis.spark
+
+// TODO: remove this once transform context settings can be read from Cassandra
+import com.microsoft.partnercatalyst.fortis.spark.ProjectFortis.Settings
+
+import scala.reflect.runtime.universe.TypeTag
+import com.microsoft.partnercatalyst.fortis.spark.analyzer.{Analyzer, ExtendedFortisEvent}
+import com.microsoft.partnercatalyst.fortis.spark.dba.ConfigurationManager
+import com.microsoft.partnercatalyst.fortis.spark.dto.{Analysis, FortisEvent}
+import com.microsoft.partnercatalyst.fortis.spark.streamprovider.StreamProvider
+import com.microsoft.partnercatalyst.fortis.spark.transforms.ZipModelsProvider
+import com.microsoft.partnercatalyst.fortis.spark.transforms.image.{ImageAnalysisAuth, ImageAnalyzer}
+import com.microsoft.partnercatalyst.fortis.spark.transforms.language.{LanguageDetector, LanguageDetectorAuth}
+import com.microsoft.partnercatalyst.fortis.spark.transforms.locations.{Geofence, LocationsExtractor, LocationsExtractorFactory, PlaceRecognizer}
+import com.microsoft.partnercatalyst.fortis.spark.transforms.locations.client.FeatureServiceClient
+import com.microsoft.partnercatalyst.fortis.spark.transforms.people.PeopleRecognizer
+import com.microsoft.partnercatalyst.fortis.spark.transforms.sentiment.{SentimentDetector, SentimentDetectorAuth}
+import com.microsoft.partnercatalyst.fortis.spark.transforms.topic.KeywordExtractor
+import org.apache.spark.streaming.StreamingContext
+import org.apache.spark.streaming.dstream.DStream
+
+object Pipeline {
+  def apply[T: TypeTag](name: String, analyzer: Analyzer[T], ssc: StreamingContext, streamProvider: StreamProvider, configurationManager: ConfigurationManager): Option[DStream[FortisEvent]] = {
+    val configs = configurationManager.fetchStreamConfiguration(name)
+    val sourceStream = streamProvider.buildStream[T](ssc, configs)
+
+    sourceStream.map(_.transform(rdd => {
+
+      // TODO: populate these local variables with values from global update/broadcast wrappers,
+      // which will use the ConfigurationManager (cassandra) to update themselves and (where applicable)
+      // broadcast changes across the Spark cluster.
+
+      val geofence = Geofence(north = 49.6185146245, west = -124.9578052195, south = 46.8691952854, east = -121.0945042053)
```

Comment:
I don't understand why we're instantiating these objects here instead of creating them once, wrapping them in the TransformContext that we previously used and then broadcasting the TransformContext to the workers ([example code](https://stackoverflow.com/a/36660199/3817588)). To me that seems as though it should provide all the functionality we need.